Formula/prover9.rb

```diff
@@ -0,0 +1,42 @@
+class Prover9 < Formula
+  desc "Automated theorem prover for first-order and equational logic"
+  homepage "https://www.cs.unm.edu/~mccune/prover9/"
+  url "https://www.cs.unm.edu/~mccune/prover9/download/LADR-2009-11A.tar.gz"
+
+  version "2009-11A"
+  sha256 "c32bed5807000c0b7161c276e50d9ca0af0cb248df2c1affb2f6fc02471b51d0"
+
+  def install
+    ENV.deparallelize
```

Comment:
Has this makefile parallelisation issue been reported?

Reply:
I don't know.  I did not. There is no public issue tracker.

Reply:
Could you report it please?

Reply:
I just did and included a [patch](https://gist.github.com/bwesterb/32361eb82e15db8691d6bf880caf4da8).